src/entities/catalogs/services/get-valid-catalogs-ids.ts

```
@@ -1,35 +1,27 @@
-} from "firebase/firestore";
+import { DocumentData } from "firebase-admin/firestore";
 import { unstable_noStore } from "next/cache";
 
+import { adminDb } from "~/shared/lib/firebase/admin";
 import { COLLECTION } from "~/shared/lib/firebase/collections";
-import { db } from "~/shared/lib/firebase/config";
 import { ValidMetadata } from "~/shared/types-schema/types";
 
 export async function getValidCatalogIds() {
   unstable_noStore();
   let catalogListData: ValidMetadata[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'catalogListData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)